packages/playground/blueprints/src/lib/steps/activate-plugin.spec.ts

```diff
@@ -84,4 +138,61 @@ describe('Blueprint step activatePlugin()', () => {
 
 		expect(php.fileExists(createdFilePath)).toBe(true);
 	});
+
+	it('should activate a plugin if it redirects during activation', async () => {
+		const docroot = handler.documentRoot;
+		php.writeFile(
+			`${docroot}/wp-content/plugins/test-plugin.php`,
+			`<?php
+			/**
+			 * Plugin Name: Test Plugin
+			 */
+			add_action( 'activated_plugin', function( $plugin ) {
+				if( $plugin == plugin_basename( __FILE__ ) ) {
+					wp_redirect( admin_url( 'edit.php' ) );
+					exit();
+				}
+			} );
+			`
+		);
+		await expect(
+			activatePlugin(php, {
+				pluginPath: 'test-plugin.php',
+			})
+		).resolves.toBeUndefined();
+	});
+
+	it('should activate a plugin if it produces a output during activation', async () => {
```

Comment:
Wait, this seems to contradict the other comment:

```
	 * Relying on the plugin activation response is not reliable because if the plugin activation
	 * produces any output, WordPress will assume it's an activation error.
	 * See WordPress source code for more details:
```

So is output an activation error, or is it okay?

Reply:
The comment is just incomplete. I explained now better, why a WP_Error during plugin activation doesn't mean the plugin isn't active. [0c09fc0](https://github.com/WordPress/wordpress-playground/pull/2066/commits/0c09fc0029bd593802b59c1a0b3b6dd6f9be9bda) There is also a section about this in the PR description with a screenshot.